addons/mail/tests/test_mail_template.py

```diff
@@ -160,3 +167,32 @@ def test_mail_template_acl_translation(self):
         # Employee without mail_template_editor group cannot create dynamic translation for mail.render.mixin
         with self.assertRaises(AccessError):
             Translation.with_user(self.user_employee).create(subject_translation_vals)
+
+    def test_mail_template_reset(self):
```

Comment:
Would make its own class, like you did for SMS templates (and we can add a unit test for the wizard in that class)